app/room.py

```diff
@@ -0,0 +1,116 @@
+import random
+from os import sys, path
+
+from app.fellow import Fellow
+from app.staff import Staff
+from app.person import Person
+
+sys.path.append(path.dirname(path.dirname(
+    path.dirname(path.abspath(__file__)))))
+
+
+class Room():
+
+    total_number_of_rooms = 0
+    room_list = {}
+
+    def __init__(self, room_name):
+        self.name = room_name.capitalize()
+        self.is_full = False
+        self.room_members = []
+        Room.total_number_of_rooms += 1
+
+    def get_room_name(self):
+        return self.name
+
+    def set_room_name(self, new_room_name):
+        if not new_room_name:
+            return "Room name can't be empty"
+        self.name = new_room_name.capitalize()
+
+    def add_room_members(self, person):
+        if not isinstance(person, Person):
+            print("Only a person, staff, or fellow instance can be added")
+            return False
+        self.room_members.append(person)
+        if len(self.room_members) == self.maximum_capacity:
+            self.is_full = True
```

Comment:
my opinion is that we don't need a dedicated boolean variable to tell us if the room is full because we can always get this boolean from comparing `len(self.room_members) == self.maximum_capacity`